tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["publicAuthority"]}],
+        }
+    ]
+    output = transforms.run_transforms({}, releases, "1")
+    assert output["parties"] == releases[0]["parties"]
+
+
+def test_public_authority_role():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["publicAuthority"]}, {"id": "2", "roles": ["publicAuthority"]}],
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.public_authority_role])
+    assert output["parties"] == releases[0]["parties"]
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["publicAuthority"]}, {"id": "2", "roles": ["buyer"]}],
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.public_authority_role])
+    assert len(output["parties"]) == 1
+
+
+def test_buyer_role():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["buyer"]}],
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.buyer_role])
+
+    assert "publicAuthority" in output["parties"][0]["roles"]
+    assert "buyer" in output["parties"][0]["roles"]
+
+
+def test_sector():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"project": {"sector": "a"}},
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.sector])
+    assert output["sector"] == "a"
+
+
+def test_additional_classifications():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"project": {"additionalClassifications": [{"scheme": "a", "id": "1"}]}},
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.additional_classifications])
+    assert output["additionalClassifications"] == [{"scheme": "a", "id": "1"}]
+
+
+def test_title():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"project": {"title": "a title"}},
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.title])
+    assert output["title"] == "a title"
```

Comment:
Is there a reason to hardcode the correct output? Rather than referencing the input, as done in `test_run_all`, e.g.
```python
assert output["title"] == releases[0]['planning']['project']['title']
```

Also, how does the transform handle when there are multiple contracting processes each with a different value in `planning/project/title`?